book/src/integrations/vscode.md

```diff
@@ -0,0 +1,10 @@
+# VScode Extension
```

Comment:
```suggestion
# VSCode Extension
```

Reply:
Hmmm... They call it "VS Code" on https://code.visualstudio.com/. But that has whitespace. Let's go with "VSCode"

Reply:
Yes whitespace would be fine! (But the `C` is still capitalized...)